### ISSUE TITLE
El menú no se despliega en pantallas pequeñas

### ISSUE THREAD

**Issue by numeroteca** [cadaveresinmob/c_inmobiliarios#70]
En mi móvil, y replicando el experimento con firefox en el navegador de mi portátil, cuando se hace click en las tres líneas horizontales en la barra del menú, no se despliega ningún menú.

**Comment by ccamara** [comment]
Efectivamente. ME temo que se trata de una versió de jquery update incompatible con el módulo de traducción y el menú desplegable. Tendré que mirarlo en cuanto pueda.
